RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.13.1
+
+- improved: formatting of the `config` section in the sphinx documentation
```

Comment:
Actually... I think this one can go into 3.13.0 as well, all to be released on Monday Nov 7th